packages/amplication-client/src/Components/FeatureIndicatorContainer.tsx

```diff
@@ -101,7 +101,15 @@ export const FeatureIndicatorContainer: FC<Props> = ({
     }
 
     return featureText;
-  }, [disabled]);
+  }, [disabled, featureText, limitationText]);
```

Comment:
Why add the featureText and the limitationText if it comes from the props?

Reply:
the eslint rule told me to add it :)